nntrainer/compiler/tflite_interpreter.cpp

```diff
@@ -391,6 +397,73 @@ TfOpNodes buildOpNodes(const GraphRepresentation &representation,
       tf_node->weightReorder(node_count);
     }
 
+    if (tf_node->getOpType() ==
+          tflite::BuiltinOperator::BuiltinOperator_CONV_2D &&
+        nodes.at(node_count + 1).get()->getOpType() ==
+          tflite::BuiltinOperator::BuiltinOperator_MUL &&
+        nodes.at(node_count + 2).get()->getOpType() ==
+          tflite::BuiltinOperator::BuiltinOperator_RELU) {
```

Comment:
Is it possible to apply this fused op for sigmoid? If it is, it might be better to refactor the codes for more general cases.

Reply:
Yes. it possible and Conv + MUL + Activation is method that tflite fusing